_maps/04-duan.md

```diff
@@ -0,0 +1,7 @@
+---
+day: 4
+creator: Anna Duan
+title: Philadelphia Fatal Car Crashes
+thumbnail: /assets/thumbnails/04-duan
+source: NA - sorry! I made this a long time ago and lost the code...
```

Comment:
**suggestion**: Looks like you left the extension off of the thumbnail image.

And no worries about the lack of source URL! I'm going to empty out the field, so that Jekyll doesn't get confused.

```suggestion
thumbnail: /assets/thumbnails/04-duan.png
source: ""
```